fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -529,6 +530,18 @@ public class Config extends ConfigBase {
     @ConfField(mutable = true)
     public static int thrift_rpc_retry_times = 3;
 
+    /**
+     * The thrift server max worker threads
+     */
+    @ConfField
+    public static int thrift_server_max_worker_threads = 4096;
```

Comment:
How many threads to set?
The user does not have the knowledge to set it.
And it must be restarted. We should avoid the restart.

Reply:
It's a stupid solution. Have you think the problem seriously?

Reply:
I just reset the location of this config.